packages/react-aria-components/src/Virtualizer.tsx

```diff
@@ -102,7 +102,9 @@ function CollectionRoot({collection, persistedKeys, scrollRef, renderDropIndicat
     onScrollEnd: state.endScrolling
   }, scrollRef!);
 
-  if (state.contentSize.area === 0) {
+  // TODO: wull have to update this when multi section loading is implemented, will need to check if all items in a collection are loaders instead perhaps
+  let hasLoadingSentinel = collection.size === 1 && collection.getItem(collection.getFirstKey()!)!.type === 'loader';
+  if (state.contentSize.area === 0 && !hasLoadingSentinel) {
```

Comment:
This will change with https://github.com/adobe/react-spectrum/pull/8110/files#diff-6026043a183382ab1099d5cc5c47d7028cef57057d81b9dfc55acfba173e6d0f, can remove this logic then